accounts-db/src/accounts.rs

```diff
@@ -552,31 +569,52 @@ impl Accounts {
         account_locks: &mut AccountLocks,
         writable_keys: Vec<&Pubkey>,
         readonly_keys: Vec<&Pubkey>,
-    ) -> Result<()> {
+        entry_accts_lookup: &mut EntryAcctLocks,
```

Comment:
nit: variable name and type isn't consistent. how about `entry_account_locks`?

Reply:
imo, better as `batch_account_locks`. `Entry` is a concept at a higher level (PoH) and really shouldn't be making its' way down here.

We have a batch of transactions, they may be **from** and entry if this is on block-verification, or they may have chance to **become** an entry if this is block-production.